app/core/api/course-instances.coffee

```diff
@@ -0,0 +1,9 @@
+fetchJson = require './fetch-json'
+
+module.exports = {
+  getByHandle: (courseInstanceID, options={}) ->
+    fetchJson("/db/course_instance/#{courseInstanceID}", options)
+
+  fetchProjectGallery: ({ courseInstanceID }, options={}) ->
+    fetchJson("/db/course_instance/#{courseInstanceID}/peer-projects", options)
```

Comment:
Yeah, agreed on consistency. Making them all take objects feels the nicest for minimizing confusion when changing code.

Another thing we'll have to normalize, though, is using `thingID` or `thingId`. I like `thingID` more (also, `userID` is user much more than `userId`)